.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: Android CI
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - '*'
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+
+      - name: Checkout submodule
+        run: git submodule sync --recursive && git submodule update --init --recursive
+
+      - name: Cache gradle and sdk
```

Comment:
I'm never 100% what we gain by caching dependencies. From what I understand, this downloads from a Google datacenter instead of a Sonatype one but might not be always much faster while opening the door to cache inconsitencies. That being said, I don't have a strong argument against it so 👍 .

Reply:
I've made the test without the cache, and actually it's a bit better.
So let's remove it